### PR TITLE
Add fugu 1.1.1

### PR DESCRIPTION
**Pull request by neronplex**

### CODE REVIEW

File: Library/Formula/fugu.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Fugu < Formula
+  homepage "https://github.com/mattes/fugu"
```
Comment (bfontaine) [inline Library/Formula/fugu.rb]: Please add a short description using `desc` above this line.

File: Library/Formula/fugu.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://github.com/mattes/fugu/releases/download/v1.1.1/fugu.v1.1.1.darwin.x86_64.tar.gz"
+  version "1.1.1"
+  sha256 "093b4f0d52c827061fa012e0242ac38fba32174ec666f5ba5d5d3ca70821d99d"
+
+  def install
+    bin.install "fugu.v1.1.1.darwin.x86_64" => "fugu"
```
Comment (bfontaine) [inline Library/Formula/fugu.rb]: Can’t we build it from source?

Comment (mikemcquaid) [inline Library/Formula/fugu.rb]: If not it'll need to be in `homebrew-binary`

File: Library/Formula/fugu.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+  test do
+    assert_equal "1.1.1\n", pipe_output("fugu --version")
```
Comment (mikemcquaid) [inline Library/Formula/fugu.rb]: Could you try and open a pull request? This should help and I'm happy to walk you through anything else: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged

Thanks!

**Comment by neronplex** [comment]
I think I want to again PR in the `homebrew-binary`. Thank you for taking the time.

**Pull request closed without merging**